client/my-sites/sharing/style.scss

```diff
@@ -62,6 +62,21 @@
 			margin-top: 12px;
 		}
 	}
+
+	.sharing-service__notice {
```

Comment:
Anything we could reuse that would allow us to avoid having to declare additional styles here? Like nesting a notice component for example?

Reply:
Used `Notice` component. Good call. Still adding this class to properly position the component since it's misaligned otherwise.